sonar-db/src/main/java/org/sonar/db/organization/OrganizationDto.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.db.organization;
+
+import javax.annotation.Nullable;
+
+public class OrganizationDto {
+  /** Technical unique identifier, can't be null */
+  private String uuid;
+  /** Functional unique identifier, can't be null */
+  private String key;
+  /** Name, can't be null */
+  private String name;
+  /** description can be null */
+  private String description;
+  /** url can be null */
+  private String url;
+  /** avatar url can be null */
+  private String avatarUrl;
+  private Long createdAt;
```

Comment:
yeah... but I'll rather have an update or insert failure because user code did not set these fields than insert 0 as a date without notice
it's safer this way